languages.toml

```diff
@@ -69,6 +74,11 @@ comment-token = "//"
 
 language-server = { command = "clangd" }
 indent = { tab-width = 2, unit = "  " }
+debug-adapter = { name = "lldb", transport = "tcp", command = "lldb-vscode", args = [], port-arg = "-p {}" }
+
+[[language.debug-configs]]
```

Comment:
Let's make the config like this:

```
debugger = {
 name = "lldb",
 ....
 templates = [ { console = "internalConsole", program = "main"}...]
```

 so rename `debug-adapter` -> `debugger` and make the configs a subkey of the adapter